src/utils/gwhisperUtils.cpp

```diff
@@ -24,5 +45,41 @@ namespace gwhisper
                 //std::cerr << "File not found at: " << f_path << std::endl;
             }
         }
+
+        void createFile(const std::string f_filePath){ //Lieber nur filePath? (inkl FileName)
+            
+
+            if(std::filesystem::exists(f_filePath)){
+                // find last . // What if no dot?
+                // Insert "_copy" before dot
+                // filepath = modified name
+
+            }
+            else{
+                std::ofstream newFile(f_filePath);
+                newFile.close();
+            }
+
+            //Todo: What if creating file fails?
```

Comment:
would it be possible to continue (only writing into ram?) then we could silently ignore the error and work in "fallback mode"
Then this function would need to return some error, which would need handling in calling code.
I think we need some "--doNotUseDescriptorCache" flag anyway to allow skipping of cache.
However alternatively we could also print an error message "Failed to create descriptor cache. Error message: ...., If you do not want to use Cache add the --doNotUseCache option"